core/includes/theme.inc

```diff
@@ -1254,8 +1254,9 @@ function path_to_theme() {
  * @param $prefixes
  *   An array of prefixes to test, in reverse order of importance.
  *
- * @return $implementations
- *   The functions found, suitable for returning from hook_theme;
+ * @return array
+ *   An array of the functions found, suitable for returning from hook_theme. 
```

Comment:
Should not *An array of the retrieved functions, suitable for being returned from hook_theme().* be better?

Reply:
I left this as-is so the description fits on one line.